recipients_app/lib/view/error_localization_helper.dart

```diff
@@ -1,7 +1,19 @@
+import "package:cloud_functions/cloud_functions.dart";
 import "package:firebase_auth/firebase_auth.dart";
 import "package:flutter_gen/gen_l10n/app_localizations.dart";
 
 String localizeExceptionMessage(Exception? ex, AppLocalizations localizations) {
+  if (ex is FirebaseFunctionsException) {
```

Comment:
okay I forgot we added propper error handling to firebase exceptions already. maybe we can just add the auth exceptions here